oracle-lib/README.md

```diff
@@ -16,17 +16,19 @@ project.
 
 ### Manually installing the Oracle JDBC Jars
 
-1. Download oracle Jars from [Oracle's website.](https://www.oracle.com/technetwork/apps-tech/jdbc-112010-090769.html) You need to download the following:
+Download oracle Jars from Oracle's website. You can usually google for each of the jar files with a search term such as "oracle ucp jar" to find the location of each jar file. You need to download the following:
```

Comment:
I had originally hoped to avoid having to tell people to 'just google it' - does that link not work anymore?